physics/module_sf_noahmp_glacier.F90

```diff
@@ -1583,7 +1583,7 @@ subroutine sfcdif1_glacier(iter   ,zlvl ,zpd    ,z0h    ,z0m , & !in
 #endif
 
 ! outputs
-    real (kind=kind_phys),                intent(out) :: fv     !< friction velocity (m/s)
+    real (kind=kind_phys),              intent(inout) :: fv     !< friction velocity (m/s)
```

Comment:
Since you're making `fv` intent(inout), please go through the code and make sure that the comments are consistent with this. I saw some places where it's still labeled as `out` only.